streaming_to_playlist.py

```diff
@@ -33,42 +36,24 @@ class StreamingToPlaylist(SpotifyHandler):
     """
 
     STREAMING_HISTORY_FILE_PREFIX = "StreamingHistory"
-    IS_PUBLIC_PLAYLIST = True
 
-    def __init__(self, spotify_yaml_path, spotify_data_path, playlist_name, playlist_description, start_time, end_time):
+    def __init__(self, spotify_yaml_path, playlist_name, playlist_description, start_time, end_time,
+                 spotify_data_path=None):
         """
         see the parse_args function for documentation on the parameters.
         """
         self._spotify_data_path = spotify_data_path
```

Comment:
move the self._spotify_data_path = spotify_data_path to the end as now it is the last parameter

Reply:
Maybe you should add a comment about what it means to not use the spotify_data_path parameter - reading directly from the API which only gives 50 last results and is not well supported.